.github/workflows/ci-cd.yml

```diff
@@ -444,14 +449,19 @@ jobs:
       uses: actions/checkout@v4
       with:
         submodules: true
+    - name: Merge distribution artifacts
+      uses: actions/upload-artifact/merge@v4
+      with:
+        name: dist
+        pattern: dist-*
```

Comment:
This is optional, it's to ensure that the same outputs are outputted in case you use these artifacts for something else outside this workflow which I'm unsure about.

Reply:
Let's exclude this for now. They aren't used elsewhere. Maybe, submit it as a separate PR later. Also, if this is added, shouldn't the step below just access `dist` instead?

Reply:
Yeah, I removed it for now since it was only meant to be backwards compatible, which is unneeded.